CMakeLists.txt

```diff
@@ -260,7 +260,7 @@ add_definitions("-DHIP_COMPILER_FLAGS=${HIP_COMPILER_FLAGS}")
 # HIP
 if( MIOPEN_BACKEND STREQUAL "HIP" OR MIOPEN_BACKEND STREQUAL "HIPOC" OR MIOPEN_BACKEND STREQUAL "HIPNOGPU")
     if(MIOPEN_USE_COMPOSABLEKERNEL)
-        find_package(composable_kernel 1.0.0 COMPONENTS device_operations)
+        find_package(composable_kernel 1.0.0 COMPONENTS device_other_operations device_gemm_operations device_conv_operations device_contraction_operations device_reduction_operations)
```

Comment:
@illsilin This new feature of the split libraries is really nice! One concern: is there any dependency among libraries or they are totally exclusive?

Reply:
In addition to @iq136boy comment above, here is the ref: https://github.com/ROCmSoftwarePlatform/composable_kernel/issues/1053

Reply:
Yes, the library will be split in just a few blocks by most common operations: gemms, convolutions, reductions, multi-head-attention, contractions, and everything else. So if your application only requires convolutions, you won't have to link all the libraries, just pick the convolutions one. If you also need softmax or layernorm, just add the "device_other_operations". 
This is nothing new, intel mkl libraries, for example, have been organized in a similar way.